imbi/templates/openapi.yaml

```diff
@@ -5427,6 +5427,102 @@ paths:
         required: true
         schema:
           type: number
+  '/projects/{id}/components':
+    parameters:
+      - name: id
+        in: path
+        required: true
+        schema:
+          type: integer
+    get:
+      summary: Retrieve project components
+      description: Retrieve the set of components that a project uses
+      tags:
+        - Components
+        - Projects
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
```

Comment:
msgpack?

Reply:
Addressed in 62524062a77cbb587655caa3f0ffbb4f00a7a7c4 and https://github.com/AWeber-Imbi/imbi-openapi/pull/26/commits/0d9a80f6ac7ff1897e991710ede92f721a84dd36